package.json

```
@@ -1,10 +1,16 @@
 {
   "scripts": {
     "test": "jest --coverage",
+    "ui-test": "jest test/js --testPathIgnorePatterns=scp-api.test.js",
     "testDebug": "node --inspect-brk node_modules/.bin/jest --runInBand",
     "lint": "eslint app/javascript",
     "fix-styles": "prettier-stylelint --write 'src/**/*.{css,scss}' "
   },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm test"
```

Is there a reason to use `npm` locally and `yarn` in CI?  I personally prefer npm over yarn, but I prefer consistency even more.  

So, unless there's a reason I'm missing, for consistency I'd propose we either A) switch to `yarn test` here, or B) use `npm ui-test` in CI (and use npm instead of yarn everywhere else).

```suggestion
      "pre-push": "yarn test"
```

---

I've just been using npm. I've updated it to yarn 0e23002